package.json

```diff
@@ -32,6 +32,7 @@
     "react": "18.2.0",
     "react-calendar": "^4.3.0",
     "react-dom": "18.2.0",
+    "react-router-dom": "^6.13.0",
```

Comment:
please run `npm uninstall react-router-dom` since we're not using it. I removed it before but looks like you may have still had it in your branch

Reply:
Done, thank you for catching this!